tests/unit/core/test_metrics.py

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: Apache-2.0
+
+import testtools
+from bandit.core.metrics import Metrics
+
+
+class BanditCoreMetricsTests(testtools.TestCase):
+
+    def test_begin_begins_collection(self):
+        metrics = Metrics()
+        fname = "test"
+        expected = {'loc': 0, 'nosec': 0}
+
+        metrics.begin(fname)  # act
```

Comment:
What does "act" stand for?

Reply:
Was using https://pypi.org/project/flake8-aaa/ - I've removed those.